src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -465,8 +465,16 @@ else if (groupIdx > lastGroup) {
      * @return true if imports group are separated internally.
      */
     private boolean checkSeparatorInGroup(int groupIdx, boolean isStatic, int line) {
-        return !beforeFirstImport && separated && groupIdx == lastGroup
-                && isStatic == lastImportStatic && line - lastImportLine > 1;
+        final boolean isStaticImportGroup =
```

Comment:
variable name seems weird as it gets it's values from nothing related to static import groups.
Is there a better name or just inline it in `inSameGroup`?

Reply:
Done.